lib/libc/picolibc/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library()
+zephyr_library_sources(libc-hooks.c)
+
+if(CONFIG_PICOLIBC AND NOT CONFIG_PICOLIBC_USE_MODULE)
+
+  # Use picolibc provided with the toolchain
+
+  zephyr_compile_options(-D_POSIX_C_SOURCE=200809 --specs=picolibc.specs)
```

Comment:
For `-D_POSIX_C_SOURCE=200809`:
```
zephyr_compile_definitions(_POSIX_C_SOURCE=200809)
```

For `--specs=picolibc.specs`:
```
  zephyr_link_libraries(-specs=picolibc.specs)
  zephyr_compile_options(-specs=picolibc.specs)
```

Reply:
Thanks, fixed.

Reply:
It looks like this change was not pushed.